### PR TITLE
Add openscap doc for report downloads and running scans from UI

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: plugins/foreman_openscap/0.6/index.md [hunk 1]
```diff
@@ -248,10 +248,14 @@ You can assign a policy in two ways:
...
-Clicking on "View Report" will lead you to the actual security audit report, with detailed information on the host's security check and suggested remediation.
+
+Report page gives information about individual rules that were checked during scan. Here you can download the actual report generated by OpenSCAP in HTML or as XML in bzip archive.
+![View ARF report]({{page.images}}/view_arf_report.png)
+
+Clicking on "View full report" at the top of the page will lead you to the actual security audit report generated by OpenSCAP, with detailed information on the host's security check and suggested remediation.
 ![ARF report]({{page.images}}/arf_report.png)
 
-## 4.5 Tailoring files ( foreman_openscap >= 0.7 )
+## 4.5 Tailoring files ( foreman_openscap >= 0.6.5 )
```
Comment (ares) [inline plugins/foreman_openscap/0.6/index.md:258]: :clap:

File: plugins/foreman_openscap/0.6/index.md [hunk 2]
```diff
@@ -276,12 +280,18 @@ Go to 'SCAP Content' tab
 Select a Tailoring file from a dropdown and then a profile that comes with it
 ![Tailoring a Policy]({{page.images}}/policy_tailoring.png)
 
+### 4.5.4 Running a scan from UI ( foreman_openscap >= 0.6.5 )
+
+In version 0.6.5 and higher, you can initiate scans from UI. Simply go to hosts page and select "Run OpenSCAP scan" from the dropdown menu. This will initiate a scan for all policies that are assigned to the host. foreman_remote_execution plugin needs to be installed for this feature to be enabled.
```
Comment (ares) [inline plugins/foreman_openscap/0.6/index.md]: > foreman_remote_execution plugin needs to be installed

maybe mentioning the version would be useful, it's currently in master only but next rex version should be 1.2.3, I kindly ask @adamruzicka to confirm the next version number

**Review by ares — CHANGES_REQUESTED** [review]
one nit otherwise big :+1:

File: plugins/foreman_openscap/0.6/index.md (hunk 2 above)
Comment (adamruzicka) [reply plugins/foreman_openscap/0.6/index.md]: Yup, the next REX release will be 1.2.3 and will contain the needed features.

**Comment by xprazak2** [comment]
I added the needed rex version,

**Review by ares — APPROVED** [review]

**Comment by ares** [comment]
Thank you @xprazak2, merging!

**Pull request merged**